src/ResourceManager/PowerBIEmbedded/Commands.Management.PowerBIEmbedded/help/AzureRM.PowerBIEmbedded.md

```diff
@@ -31,24 +31,3 @@ Tests the existence of an instance of PowerBI Embedded Capacity.
 
 ### [Update-AzureRmPowerBIEmbeddedCapacity](Update-AzureRmPowerBIEmbeddedCapacity.md)
 Modifies an instance of PowerBI Embedded Capacity.
-
-
-## PowerBI Workspace Collection Cmdlets
-### [Get-AzureRmPowerBIWorkspace](Get-AzureRmPowerBIWorkspace.md)
```

Comment:
@selasagi shouldn't you be removing the `EmbeddedCapacity` links above since they are the ones that aren't working?

Reply:
On the beginning the EmbeddedCapacity links not appeared so I added them and now (when I saw that we can't separate the 2 resources links to two sections) I decided to remove the links to the WorkspaceCollection (the old  resource)